sdk/python/kubeflow/trainer/hf_llm_training.py

```diff
@@ -177,9 +187,14 @@ def parse_arguments():
     transformer_type = getattr(transformers, args.transformer_type)
 
     logger.info("Setup model and tokenizer")
-    model, tokenizer = setup_model_and_tokenizer(
-        args.model_uri, transformer_type, args.model_dir
-    )
+    if args.num_labels == "None":
+        model, tokenizer = setup_model_and_tokenizer(
+            args.model_uri, transformer_type, args.model_dir, 0
+        )
+    else:
+        model, tokenizer = setup_model_and_tokenizer(
+            args.model_uri, transformer_type, args.model_dir, int(args.num_labels)
+        )
```

Comment:
I think, you can just set the default value for num_labels in the parser as None, and then check if `num_labels=None` in the setup_model_and_tokenizer function.

Reply:
@andreyvelich Thank you for your suggestions! I've made the updates. Please review them and let me know if there are any issues.